### PR TITLE
Feature/review form component

### PR DESCRIPTION
**Pull request by stephanie-roe**
# Description
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 

- Structured the Review Form component and integrated API to hold user reviews. 

Were there any issues that arose? 
- Difficulty fetching by ID

Is there anything you need from your teammates? 
-n/a

Any other comments, questions, or concerns?
-n/a

## Issues
Closes #

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have deleted all comments after functionality and understanding is final.
- [ ] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Components/ReviewForm.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+class ReviewForm extends React.Component<ReviewFormProps, ReviewFormState> {
+    state: ReviewFormState = {
+        name: "",
+        content: "",
+        reviews: []
+    }
+
+    componentDidMount() {
```
Comment (userigorgithub) [inline src/Components/ReviewForm.tsx:29]: Great job fetching new api's data!

File: src/Components/ReviewForm.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+type ReviewFormProps = {
+    id: string
+}
+
+type ReviewFormState = {
+    reviews: ReviewObject[],
+    name: string,
+    content: string,
+    // do we need id here?
```
Comment (aominhlong) [inline src/Components/ReviewForm.tsx:13]: We can remove any unnecessary comments.

File: src/Components/ReviewForm.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        console.log(this.props.id)
+        fetch(`http://localhost:3001/api/v1/reviews`)
+        .then(response => {
+            if (response.ok) {
+                return response.json()
+            } else {
+                throw Error(response.statusText)
+            }
+        })
+        .then(data => {
```
Comment (userigorgithub) [inline src/Components/ReviewForm.tsx:39]: Hmm, a little confused why we need filtering here... may be instructors know?

**Review by userigorgithub — APPROVED** [review]
Good job team working on implementing comments!

**Review by aominhlong — APPROVED** [review]

**Pull request merged**